pkgs/development/python-modules/jaxlib/default.nix

```diff
@@ -302,13 +397,13 @@ let
     inherit meta;
   };
   platformTag =
-    if stdenv.hostPlatform.isLinux then
+    if effectiveStdenv.hostPlatform.isLinux then
       "manylinux2014_${arch}"
-    else if stdenv.system == "x86_64-darwin" then
+    else if effectiveStdenv.system == "x86_64-darwin" then
       "macosx_10_9_${arch}"
-    else if stdenv.system == "aarch64-darwin" then
+    else if effectiveStdenv.system == "aarch64-darwin" then
       "macosx_11_0_${arch}"
-    else throw "Unsupported target platform: ${stdenv.hostPlatform}";
+    else throw "Unsupported target platform: ${effectiveStdenv.hostPlatform}";
```

Comment:
(TODO for myself: look into and maybe open a separate issue)

Out of scope, but I didn't notice this before and it looks suspicious that a downstream derivation is computing the python "platform tags", and the more so that it uses `throw`. I wonder the nixpkgs python machinery is already implementing this logic elsewhere